lib/ruby_lsp/listeners/hover.rb

```diff
@@ -193,6 +200,18 @@ def generate_heredoc_hover(node)
         end
       end
 
+      sig { params(keyword: String).void }
+      def show_keyword_documentation(keyword)
```

Comment:
```suggestion
      def handle_keyword_documentation(keyword)
```
(for consistency with other methods in `Hover`).